File: server/admin-server/Controllers/Helpers/Credentials.cs
using System;

namespace Microarea.AdminServer.Controllers.Helpers
{
	//================================================================================
	public class Credentials
	{
		public string AccountName;
		public string Password;
	}

	//================================================================================
	public class DatabaseCredentials
	{
		public string Provider;
		public string Server;
		public string Database;
		public string Login;
		public string Password;

		//-----------------------------------------------------------------------------	
		public bool Validate()
		{
			return !string.IsNullOrWhiteSpace(Provider) && !string.IsNullOrWhiteSpace(Server) && !string.IsNullOrWhiteSpace(Database) && !string.IsNullOrWhiteSpace(Login);
		}
	}

	//================================================================================
	public class AccountModification
    {
        public string AccountName;
		public string InstanceKey;
        public int Ticks;
        //-----------------------------------------------------------------------------	
        public AccountModification(string accountName, string instanceKey, int ticks)
        {
            this.AccountName = accountName;
			this.InstanceKey = instanceKey;
            this.Ticks = ticks;
        }
    }
}

File: client/apps/console/src/app/authentication/credentials.ts
import { SubscriptionDatabase } from './../model/subscriptionDatabase';

//================================================================================
export class Credentials {
    
    accountName: string;
    password: string;
    
    constructor() {
        this.accountName = '';
        this.password = '';
    }
}

//================================================================================
export class DatabaseCredentials {
    
    Provider: string;
    Server: string;
    Database: string;
    Login: string;
    Password: string;
    
    
    constructor() {
        this.Provider = '';
        this.Server = '';
        this.Database = '';
        this.Login = '';
        this.Password = '';
    }
}

//================================================================================
export class ChangePasswordInfo {
    
  AccountName: string;
  Password: string;
  NewPassword: string;
  Temporary: boolean;
  
  constructor() {
      this.AccountName = '';
      this.Password = '';
      this.NewPassword = '';
      this.Temporary = false;
  }
}

